packages/core/src/tracing/instrumentation/process/process.js

```diff
@@ -24,6 +24,10 @@ exports.init = function init(config) {
   if (config.tracing.activateBullProcessInstrumentation) {
     shimmer.wrap(process, 'emit', shimProcessEmitForBullChildWorker);
     shimmer.wrap(process, 'send', shimProcessSendForBullChildWorker);
+
+    // Activate immediately, otherwise we miss the first message event and the instana
+    // headers would not get removed from the message.
+    exports.activate();
```

Comment:
Instead of activating everything, we just activate what we need. The config even says it

> activateBullProcessInstrumentation